index.Rmd

```diff
@@ -27,6 +27,7 @@ I loaded required packages
 
 ```{r, message=F, warning=F}
 library(raster)
+library(threejs)
```

Comment:
Call all packages at beginning of your code so that others may see what packages they need without searching through the rest of your code.